### PR TITLE
Updated README.md

### LINKED ISSUE #9: Want to update README.md file
**Is your feature request related to a problem? Please describe.**
Want to add How to Contribute section and Contributors profiles into README.md file

@sbplat can you assign this to me please?

### PR DESCRIPTION
**Pull request by sb-decoder**
# Description

I added How to Contribute Section and  contributors profile images into main readme file and fixed issue: #9 

Fixes #9 

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)
- [x] Documentation (non-breaking change that updates the documentation)

# Checklist:

- [x] I have read the [Contribution Guidelines](https://github.com/sbplat/Macro-API/blob/main/CONTRIBUTING.md) and [Code of Conduct](https://github.com/sbplat/Macro-API/blob/main/CODE_OF_CONDUCT.md)
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

**Review by sbplat — CHANGES_REQUESTED** [review]
Could you move the contributing guide over to `CONTRIBUTING.md` and make a hyperlink that links over to the `CONTRIBUTING.md` file

File: README.md [hunk 1]
```diff
@@ -38,6 +38,37 @@ mkdir build && cd build
 cmake ..
 cmake --build .
 ```
+## How to contribute:
+
+- Drop a :star: on the Github repository (optional)<br/>
+
+- Before Contribute Please read [CONTRIBUTING.md](https://github.com/sbplat/Macro-API/blob/main/CONTRIBUTING.md) and [CODE_OF_CONDUCT.md](https://github.com/sbplat/Macro-API/blob/main/CODE_OF_CONDUCT.md)
```
Comment (sbplat) [inline README.md]: grammer

File: README.md [hunk 2]
```diff
@@ -38,6 +38,37 @@ mkdir build && cd build
 cmake ..
 cmake --build .
 ```
+## How to contribute:
+
+- Drop a :star: on the Github repository (optional)<br/>
```
Comment (sbplat) [inline README.md]: I don't think line breaks (`br`) are necessary

File: README.md [hunk 3]
```diff
@@ -38,6 +38,37 @@ mkdir build && cd build
...
+
+- Before Contribute Please read [CONTRIBUTING.md](https://github.com/sbplat/Macro-API/blob/main/CONTRIBUTING.md) and [CODE_OF_CONDUCT.md](https://github.com/sbplat/Macro-API/blob/main/CODE_OF_CONDUCT.md)
+
+- Create an issue of the project or a feature you would like to add in the project and get the task assigned for youself.(Issue can be any bug fixes or any feature you want to add in this project).
```
Comment (sbplat) [inline README.md]: typo

File: README.md [hunk 4]
```diff
@@ -38,6 +38,37 @@ mkdir build && cd build
 cmake ..
 cmake --build .
 ```
+## How to contribute:
```
Comment (sbplat) [inline README.md]: `How to contribute:` should be changed to `Contributing` for consistency.
Could you also hyperlink this from the navigation links?

**Comment by sb-decoder** [comment]
> Could you move the contributing guide over to `CONTRIBUTING.md` and make a hyperlink that links over to the `CONTRIBUTING.md` file

Please check correction and tell me if you want further changes.

File: README.md (hunk 2 above)
Comment (sb-decoder) [reply README.md]: removed!

**Comment by sbplat** [comment]
I made a few changes to both of the files. Is there anything that you want to add?

**Comment by sb-decoder** [comment]
> I made a few changes to both of the files. Is there anything that you want to add?

No it's OK.

**Comment by sbplat** [comment]
Thanks!

**Pull request merged**